decidim-core/app/models/decidim/user.rb

```diff
@@ -20,7 +21,7 @@ class User < ApplicationRecord
     has_many :identities, foreign_key: "decidim_user_id", class_name: "Decidim::Identity", dependent: :destroy
     has_many :memberships, class_name: "Decidim::UserGroupMembership", foreign_key: :decidim_user_id, dependent: :destroy
     has_many :user_groups, through: :memberships, class_name: "Decidim::UserGroup", foreign_key: :decidim_user_group_id
-    has_many :follows, foreign_key: "decidim_user_id", class_name: "Decidim::Follow", dependent: :destroy
+    has_many :following, foreign_key: "decidim_user_id", class_name: "Decidim::Follow", dependent: :destroy
```

Comment:
Is this relation used at all? I understand you renamed it to not conflict with the relations provided by `Followable` but I don't think we're using it since the rename isn't used anywhere else in the patch and it didn't break anything?

Reply:
I couldn't find either were it was used, I'll remove it